api/v1alpha1/clusterorder_types.go

```diff
@@ -63,20 +66,31 @@ const (
 	// ClusterOrderConditionControlPlaneAvailable means the cluster control plane is ready
 	ClusterOrderConditionControlPlaneAvailable ClusterOrderConditionType = "ControlPlaneAvailable"
 
-	// ClusterOrderConditionNodePoolAvailable means the node pool has the correct number of nodes
-	ClusterOrderConditionNodePoolAvailable ClusterOrderConditionType = "NodePoolAvailable"
-
 	// ClusterOrderConditionAvailable means the cluster is available
 	ClusterOrderConditionAvailable ClusterOrderConditionType = "Available"
 )
 
+// ClusterOrderClusterReferenceType contains a reference to the namespace created by this ClusterOrder
+type ClusterOrderClusterReferenceType struct {
+	// Namespace that contains the HostedCluster resource
+	ClusterNamespace   string `json:"clusterNamespace"`
```

Comment:
I don't think we need to repeat "cluster" here since it's already scoped by the parent field.

`instance.Status.ClusterReference.Namespace` is sufficient.